addons/point_of_sale/static/src/js/ChromeWidgets/ProxyStatus.js

```diff
@@ -18,21 +18,10 @@ odoo.define('point_of_sale.ProxyStatus', function(require) {
             this.index = 0;
         }
         mounted() {
-            this.env.pos.proxy.on('change:status', this, this._onChangeStatus);
+            this.env.proxy.on('change:status', this, this._onChangeStatus);
         }
         willUnmount() {
-            this.env.pos.proxy.off('change:status', this, this._onChangeStatus);
-        }
-        async onClick() {
```

Comment:
Why did you remove this behavior?

Reply:
I didn't. Check the change in https://github.com/odoo/odoo/pull/82461/files/fbd966a132d6dd9c12fc7892192aa5714adfd61e#diff-e0849d9d6458581f1aeb9f69fcfa358761e1b1a775c37616dcb4dca20bf4a0e1

Reply:
Or check the change in `ProxyStatus.xml`